Scientific research proposal (#336)
space-wizards/docs: src/en/space-station-14/departments/science/proposals/scientific-research-proposal.md

# Scientific research proposal

| Designers     | Implemented | GitHub Links |
|---------------|-------------|--------------|
| chromiumboy   | :x: No      | TBD          |

## Problem statement
At present the science department faces several key gameplay issues when it comes to unlocking items through scientific research. These include (but are not limited to):
-	A large number of research projects that can be considered ‘filler’. These projects unlock either very niche or very common place items on the station; a choice between unlocking magboots, hydroponic trays, or buildable research disk terminals is not compelling for the player. 

> -	Due to the high proportion of filler projects at lower research tiers, typically one project will be seen as the ‘optimal’ choice to research (generally either in the industrial or experiment disciplines), leading to many niche items being ignored in the early to mid stages of the shift when they would be most useful, as well as a sense of sameness in technology unlocking despite the research projects being presented at random.

Reviewer: ```suggestion
- High proportion of 'filler' tech at lower research tiers leads to considering such technologies as stepping stones towards some final project - some t3 tech usually. This means that picking tech does not feel really important to player, lowering involvement, giving dissatisfaction that work was done but result is not useful. This also means that techs that are low tier but belong to discipline with non desider t3 will most likely will not be researched during the early to mid game stages of the shift when they would be most useful and can be in high demand.
```

Reviewer: The idea is, kinda, that people are not having fun themselves, by spending earned points on it AND they are not letting other people have fun, because what other department want 'it is not in discipline that was targeted'. which is double negative.

> -	Many research items seem to be under costed with respect to the current ease of points generation. This leads to shifts where all research projects can be readily completed, particularly during longer than normal shifts and / or shifts that have a large research team.

Reviewer: ```suggestion
-	Many research items seem to be under costed with respect to the current ease of points generation. This leads to shifts where all research projects can be readily completed, particularly during longer than normal shifts and / or shifts that have a large research team. Most of the time sci department will have almost nothing to do with researches after that, basically having zero real endgame (toying with what you researched without any real aim is hard to consider 'endgame content' ).
```

> -	Once all research projects have been completed, there is no longer any compelling reason to conduct any further research.

Reviewer: ```suggestion
-	Once all research projects have been completed, there is no longer any compelling reason to conduct any further research, leaving department without any kind of endgame.
```

> 

Reviewer: Also i'd propose to add problem that was leading to original discussion - other departments don't really hope to get somthing very cool from science in mid-round, and endgame is not something you see every round.
I'd propose to something like that:

- Other departments are not relying on sci in any good way at all, which makes it harder to create mechanics, that require assistance from other departments for science. Early research contains most of stuff is pre-mapped stuff, or very minor improvements, not that much desired by majority of players (fancy drill is cool, but not worth staying and yelling 20 minutes on reception).

Other than that i'd propose adding words about sci hoarding resources (which is okay, i guess) and not really giving people stuff - because they consider that would be bad exchange.

- It is occasional problem that sci department is not willing to share items they researched because they feel like giving resources they collected to print items for other people for free is a bad exchange. It seems like it is related to the problem of HOW sci gets such resources - they either get them from other department  (which is not guaranteed) or from maints (as not renewable source!). Overall feeling giving up non-renewable source (from perspective of your department) feels bad.

Author: The first point is a good inclusion, but I'm not worried about the second

Reviewer: 👍

> ## Proposed changes

Reviewer: I'd propose using this 

The changes to scientific research outlined in this proposal have been devised in an attempt to alleviate the issues listed above. These changes include:
-	Making all low-tier / filler researchable items available to print at round start. These include items such as proximity sensors, TV circuit boards, and surveillance camera equipment.
-	Making the more desirable mid-tier researchable items (e.g., advanced cleaning, lasers, and high-capacity batteries) also available to print at round start, but with the caveat that science points are required as part of their construction. Lore-wise, this can be explained as the technology already being widely known, however, you need to purchase technology licenses from NT in order to print them (and NT is mostly interested in data-sets that RD server produces!).
-	Scientists can print technology licenses from the tech disk terminal, exchanging their research points for discipline specific licenses (e.g., a single civilian technology license might cost 150 research points, a single industrial technology license 250, and a single arsenal technology license 350).
-	These licenses can then be fed into (or ejected from) an appropriate lathe just like any other resource material (e.g., arsenal technology licenses can be inserted into security and ammo fabs, civilian technology licenses into medical fabs, and experimental technology licenses into protolathes). That means that each department will get its own kind of fab - med and sec have it currently, engi, cargo and service will get new ones, and sci will get fancy recolored one.
-	Licensed items require a number of licenses that correspond to their relative power / usefulness (e.g., a laser carbine might require five licenses to print, in addition to their other base materials, while a box of incendiary ammunition might only need one license). 
-	The highest tier / most powerful technologies (e.g., bluespace backpacks, speed boots, teleporter pads; perhaps a dozen technologies all up) would still need to be researched the traditional way, requiring an expenditure of research points at the R&D computer to unlock (in addition to the license cost to print them afterward; unfortunately, NT owns all patentable technology produced by their research staff). However, each of these research projects now requires a vast amount of research points to unlock, in the range of hundreds of thousands. The ultimate goal is that unlocking a new technology should be a relatively rare occurrence and a nice trading barging when trying to get something from other department. Maybe we can expected in the timeline of one shift,to have only two or three unlocks per shift being common. Major new research cost will also motivate sci to give away licenses for crafting easier - it doens't really matter much if u give away 1000-2000 sci points to janitor, if your aim is to collect, for example, 80 000 - its not comparable, it won't hurt you that much in the long run.

```mermaid
flowchart LR
	A1 -- Fixed by --> B1
	A2 -- Fixed by --> B1
	A3 -- Fixed by --> B1
	A4 -- Also fuels --> C2
	A5 -- Fixed by --> B2
	A6 -- Fixed by --> B2
	B1 -- Leads to --> C1
	B1 -- Leads to --> C2
	B2 -- Leads to --> C4
	C1 -- Fixed by --> D1
	C2 -- Fixed by --> D2
	C3 -- Fixed by --> D1
	C4 -- Fixed by --> D1
	
	subgraph Problems[baseline problems]
		A1['filler' tech <br>doesn't feels involving]
		A2['non-target' t1/t2 disciplines <br>rarely being researched <br >in early game, when <br> they are most useful]
		A3['filler' tech is obstructing <br> view of 'useful tech' ]
		A4[Important researches <br>have inadequate price]
		A5[Sci unwilling <br>to waste a lot <br>of time on reception]
		A6[Sci unwilling <br>to share hard-earned <br>hard-recoverable - for them <br>- resources]
	end
	subgraph proposed [proposed solutions]
		B1[Move 'filler' tech <br>to be available roundstart]
		B2[Each department <br>gets own 'protolathe' <br>with recepie restrictions]
	end
	subgraph intermediate[intermediate problems]
		C1[Diminishing role <br> of sci department]
		C2[Less 'content' <br>for science department]
		C3[Almost nothing to do after <br>most desired tech <br> was researched]
		C4[Why bother giving<br> something or help sci?]
	end
	subgraph final[final fixes]
		D1[Add some currency, <br>made from science points <br> to be used in most <br>of printed tech recepies]
		D2[Rise tech research <br>base costs and use dynamic <br> research cost mechanic]
	end

```

Reviewer: i will DM graph code to you in discord

Author: Thanks, that looks really cool

The changes to scientific research outlined in this proposal have been devised in an attempt to alleviate the issues listed above. These changes include:
-	Making all low-tier / filler researchable items available to print at round start. These include items such as proximity sensors, TV circuit boards, and surveillance camera equipment.

> -	Making the more desirable mid-tier researchable items (e.g., RCDs, lasers, and high-capacity batteries) also available to print at round start, but with the caveat that science points are required as part of their construction. Lore-wise, this can be explained as the technology already being widely known, however, you need to purchase technology licenses from NT in order to print them. 

Reviewer: Mentioning RCD here is strategically bad - better omit it. RCD have many very different features, and they lead to many different problems - lets just not get into that swamp of a discussion.

-	Licensed items require a number of licenses that correspond to their relative power / usefulness (e.g., a laser carbine might require five licenses to print, in addition to their other base materials, while a box of incendiary ammunition might only need one license). 
-	The highest tier / most powerful technologies (e.g., bluespace backpacks, speed boots, teleporter pads; perhaps a dozen technologies all up) would still need to be researched the traditional way, requiring an expenditure of research points at the R&D computer to unlock (in addition to the license cost to print them afterward; unfortunately, NT owns all patentable technology produced by their research staff). However, each of these research projects now requires a vast amount of research points to unlock, in the range of hundreds of thousands. The ultimate goal is that unlocking a new technology should be a relatively rare occurrence within the shift, with perhaps only two or three unlocks per shift being common.

> 

Reviewer: Its important to add, that a lot of researches are 'schema/board' only, and to roll with this new scheme those boards should be now available at respective fabs. Which leaves 'board printer' device kinda useless. We can just add every board into it, or remove it. I'm not sure, honestly, which is better.

Author: Do you mean the circuit imprinter? I mean. there's no reason why it couldn't print licensed boards as well, or just be the sole source of boards

Reviewer: Your answer does not contain any clue what EXACTLY should be line of separation - which stuff do what. At least conceptually.
I suppose you would like to leave 'circuit printer' to have just have all circuit boards as it works right now. If so - i'd propose mentioning in proposal, that boards from recepies are going to be available from respective department fabs AND ALSO from circuit printer - as it is now.

> 

Author: I wouldn't remove them

> 

Reviewer: that still leave 2 options. If you want to leave them as sci-only, then what recepies should protolathe receive? Only science? Or maybe it should be able to select department that is owning it and change to some fab, based on some action (with authorization)?

## Benefits of the proposed changes

> The proposed changes seek to shift scientific research from being a short-term and low-impact activity to a long-term goal for the shift, while still allowing the science department to outfit the crew with limited amounts of specialized high-tech equipment by exchanging their research points for technology licenses.

Reviewer: ```suggestion
The proposed changes seek to shift scientific research from being a short-term and low-impact activity to a long-term goal for the shift, while still allowing the science department to outfit the crew with limited amounts of specialized high-tech equipment by exchanging their research points for technology licenses, with additional value of possible increase in department interactions and more motivation for cooperation (value from which can now be easily calculated and converted).
```

Author: I think its fine as is

Reviewer: ok

|---------------|-------------|--------------|
| chromiumboy   | :x: No      | TBD          |

## Problem statement
At present the science department faces several key gameplay issues when it comes to unlocking items through scientific research. These include (but are not limited to):
-	A large number of research projects that can be considered ‘filler’. These projects unlock either very niche or very common place items on the station; a choice between unlocking magboots, hydroponic trays, or buildable research disk terminals is not compelling for the player. 
-	A high proportion of 'filler' tech at lower research tiers leads to players considering such technologies merely as stepping stones towards some final project (typically a specific high tier technology). This also means that useful low tier technologies that belong to disciplines outside of their target are more likely to be overlooked during the early to mid game stages of the shift when they would be most useful.
- Due to the abundance of low to mid-tier researachable items available on the station at round start, there is a limited need for other departments to request equipment from the science department

> -	Many research items seem to be under costed with respect to the current ease of points generation. This leads to shifts where all research projects can be readily completed, particularly during longer than normal shifts and / or shifts that have a large research team.

Reviewer 2: in my opinion research __should__ be completed before the end of an abnormally long shift, thats because to do a project you not only have to have researched it but then make it and some projects will require things from multiple tech trees which takes further time.

Reviewer 2: also a ccvar for a research cost multiplier might be an easy solution to make research slower.

Reviewer: there is a separate idea of https://github.com/space-wizards/docs/pull/329 but overall - nope.

I don't think flat multiplier going to do any good job, as costs are not just 'flat low', they are not even balanced agains each other very much. multipler will just make it feel worse. It looks like it needs careful rebalance :(

Author: I mean. Both is good. Yes, all the research costs have to be rebalanced, but a cvar to adjust all the final values by a multiplier is would be a good inclusion as well
